### DISCUSSION TITLE
刪除討論

### DISCUSSION THREAD

**Discussion by jeffw858** [Lidemy/forum#274] [其他問題] [answered]
![截圖 2022-07-28 下午12 51 58](https://user-images.githubusercontent.com/110077472/181625332-38df7ee4-75c9-44a9-b7bd-d79c8865a1c5.png)
我的Github側邊欄怎麼沒有刪除討論的按鍵呢？
我看Github教學有可以刪除的呀？
https://docs.github.com/cn/discussions/managing-discussions-for-your-community/managing-discussions#deleting-a-discussion

**Comment by cwc329** [comment]
這個功能應該是討論區的管理員或者有某些權限才能使用，
所以你不能刪除。

**Comment by cwc329** [comment, marked as answer]
BTW
如果以後要 PO 程式碼討論可以先看一下 [git markdown](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) 教學，
使用 code block 並且加上語言可以讓網頁自動 syntax highlight。
這樣如果別人要對你的 code 某些地方提出建議就可以複製貼上，而不是還要自己再打一次。

**Reply by jeffw858** [reply]
了解！感謝！我真的很菜
